lib/lita/handlers/totems.rb

```diff
@@ -241,11 +264,23 @@ def waiting_duration(time)
       end
 
       def yield_totem(totem, user_id, response)
+        waiting_since_hash = redis.hgetall("totem/#{totem}/waiting_since")
```

Comment:
Nit: I think we should move some of this logic into private helper methods or into a separate static class as mentioned here: https://github.com/change/lita-totems/pull/2/files#r826453118
```ruby
       waiting_since_hash = redis.hgetall("totem/#{totem}/waiting_since")
        capture_holding_time(totem, waiting_since_hash)
        redis.srem("user/#{user_id}/totems", totem)
        redis.hdel("totem/#{totem}/waiting_since", user_id)
        redis.hdel("totem/#{totem}/message", user_id)
        next_user_id = redis.lpop("totem/#{totem}/list")
        if next_user_id
          capture_waiting_time_stats(totem, waiting_since_hash)
```